### PR TITLE
Build no server oatpp

### PR DESCRIPTION
**Pull request by Bycob**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -965,84 +970,83 @@ if (USE_TENSORRT)
...
-            -DCMAKE_BUILD_TYPE=Release
-            -DBUILD_SHARED_LIBS=OFF
-            -DOATPP_BUILD_TESTS=OFF
-            -DOATPP_INSTALL=OFF
-            -DOATPP_MODULES_LOCATION=CUSTOM
-            -DOATPP_DIR_SRC=${CMAKE_BINARY_DIR}/oatpp/src/oatpp/src
-            -DOATPP_DIR_LIB=${CMAKE_BINARY_DIR}/oatpp/src/oatpp/src
-        DEPENDS oatpp
-    )
+set(OATPP_VERSION "1.2.5")
```
Comment (beniz) [inline CMakeLists.txt:984]: Beware, we had the `USE_HTTP_SERVER_OATPP` flag to be able to switch back to the older http server (when available / buildable...). We don't want to remove it just now (if only because of the oatpp bug/leak with zlib).

File: CMakeLists.txt [hunk 2]
```diff
@@ -798,20 +798,25 @@ if (USE_TORCH)
 
   set(PYTORCH_VISION_COMPLETE ${CMAKE_BINARY_DIR}/CMakeFiles/pytorch_vision-complete)
   set(TORCHVISION_LOCATION "${CMAKE_BINARY_DIR}/pytorch_vision/src/pytorch_vision-install")
+
+  set(PYTORCH_VISION_FLAGS
```
Comment (beniz) [inline CMakeLists.txt:802]: I guess this does not belong to this PR.

File: CMakeLists.txt (hunk 1 above)
Comment (Bycob) [reply CMakeLists.txt:984]: I did not remove the flag, I just always activate oatpp build, since we need it for DTOs

**Comment by Bycob** [comment]
@Mergifyio rebase

**Review by fantes — APPROVED** [review]

**Review by beniz — APPROVED** [review]
